--- sysroot-octave/ucrt64/lib/pkgconfig/UMFPACK.pc.orig	2025-11-04 18:25:50.322276312 +0100
+++ sysroot-octave/ucrt64/lib/pkgconfig/UMFPACK.pc	2025-11-04 18:25:57.282214779 +0100
@@ -2,7 +2,7 @@
 # All Rights Reserved.
 # SPDX-License-Identifier: GPL-2.0-or-later
 
-prefix=/usr
+prefix=/ucrt64
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 includedir=${prefix}/include/suitesparse
--- sysroot-octave/ucrt64/lib/pkgconfig/AMD.pc.orig	2025-11-04 18:29:23.860382837 +0100
+++ sysroot-octave/ucrt64/lib/pkgconfig/AMD.pc	2025-11-04 18:29:33.340298566 +0100
@@ -2,7 +2,7 @@
 # All Rights Reserved.
 # SPDX-License-Identifier: BSD-3-Clause
 
-prefix=/usr
+prefix=/ucrt64
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 includedir=${prefix}/include/suitesparse
--- sysroot-octave/ucrt64/lib/pkgconfig/SuiteSparse_config.pc.orig	2025-11-04 18:29:27.268352546 +0100
+++ sysroot-octave/ucrt64/lib/pkgconfig/SuiteSparse_config.pc	2025-11-04 18:29:45.880187057 +0100
@@ -2,7 +2,7 @@
 # All Rights Reserved.
 # SPDX-License-Identifier: BSD-3-clause
 
-prefix=/usr
+prefix=/ucrt64
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 includedir=${prefix}/include/suitesparse
--- sysroot-octave/ucrt64/lib/pkgconfig/openblas.pc.orig	2025-11-07 12:40:34.182221621 +0100
+++ sysroot-octave/ucrt64/lib/pkgconfig/openblas.pc	2025-11-07 12:40:48.571837307 +0100
@@ -1,8 +1,8 @@
-libdir=/usr/lib
+libdir=/ucrt64/lib
 libprefix=
 libnamesuffix=
 libsuffix=
-includedir=/usr/include
+includedir=/ucrt64/include
 omp_opt=
 openblas_config= USE_64BITINT= DYNAMIC_ARCH=1 DYNAMIC_OLDER=1 NO_CBLAS=1 NO_LAPACK=1 NO_LAPACKE=1 NO_AFFINITY=1 USE_OPENMP= GENERIC MAX_THREADS=24
 version=0.3.29
